flowey/flowey_hvlite/src/pipelines_shared/gh_flowey_bootstrap_template.yml

```diff
@@ -1,11 +1,33 @@
+# TODO: Add additional installers for all prerequisites that get skipped if
+# the programs are already installed.
+
+# Known additional prerequisites on Windows:
+#   - Git and bash, both added to path
+#   - For running VMM tests, Hyper-V and WHP must be enabled
+#   - Visual Studio Build Tools
+#     - Microsoft.VisualStudio.Component.VC.Tools.x86.x64
+#     - Microsoft.VisualStudio.Component.VC.Tools.ARM64
+#     - Microsoft.VisualStudio.Component.Windows11SDK.22621
+#     - Microsoft.VisualStudio.Component.VC.Llvm.Clang (added to path)
+
+#### Flowey Build Dependencies
+
+# On Linux, install gcc and rust to build flowey. 
+# Retry apt-get on failure in case another process has the lock
 - run: |
     set -x
+    i=0; while [ $i -lt 5 ] && ! sudo apt-get update; do let "i=i+1"; sleep 1; done;
+    sudo apt-get -o DPkg::Lock::Timeout=60 install gcc -y
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+    . "$USERPROFILE/.cargo/env"
```

Comment:
USERPROFILE is not a thing on Linux, right?

Reply:
Oops, it should be $HOME